services/shotgrid_common/utils.py

```diff
@@ -43,8 +43,6 @@ def _sg_to_ay_dict(sg_entity: dict, project_code_field=None) -> dict:
 
         if not label and not task_type:
             raise ValueError(f"Unable to parse Task {sg_entity}")
-        else:
```

Comment:
@Minkiu this was overriding the label every time

Reply:
Here I though I was playing it safe... :facepalm:  thanks!